### DISCUSSION TITLE
Multiple ROI in one image

### DISCUSSION THREAD

**Discussion by barabba2005** [emgucv/emgucv#672] [Q&A]
Hi all
I have more ROI and I would like to put them in a single image respecting the right positions.
I tried with:

if (faces.Length > 0)
                {                    
                    foreach (var face in faces)
                    {                         
                        currenteROI.ROI = face;
                        ROIresize = currenteROI.Resize(320, 180, Inter.Linear); //Fix ROI image size  
                        picDetected.Padding = new Padding(face.X, face.Y, 0, 0);                        
                    }                              
                }     
picDetected.Image = currenteROI.ToBitmap<Bgr, byte>();

but i see only one of ROI.

Thanks in advance

**Comment by barabba2005** [comment]
Probably i solved with this:

for (int R=0; R<faces.Length; R++) //Draw every ROI into picDetected
                {
                    currenteROI.ROI = faces[R];
                    ROIresize = currenteROI.Resize(100, 100, Inter.Linear);
                    imageResult.ROI = new Rectangle(faces[R].X, faces[R].Y, 100, 100); //new Rectangle
                    ROIresize.CopyTo(imageResult);
                    imageResult.ROI = Rectangle.Empty;
                }

**Reply by emgucv** [reply]
Thanks for the update.
